lib/mockbuilder.py

```diff
@@ -42,9 +42,10 @@ def __init__(self):
         self.archive = None
```

Comment:
In the longer commit msg, I'd prefer "This allows multiples processes to use Mock concurrently, creating a separate chroot for each". This makes clearer that different processes do *not* share a chroot.